rpcs3/Emu/NP/np_handler.h

```diff
@@ -35,9 +35,9 @@ class np_handler
 	static std::string ip_to_string(u32 addr);
 	static std::string ether_to_string(std::array<u8, 6>& ether);
 	// Helpers for setting various structures from string
-	static void string_to_npid(const char* str, SceNpId* npid);
-	static void string_to_online_name(const char* str, SceNpOnlineName* online_name);
-	static void string_to_avatar_url(const char* str, SceNpAvatarUrl* avatar_url);
+	static void string_to_npid(const std::string&, SceNpId* npid);
+	static void string_to_online_name(const std::string&, SceNpOnlineName* online_name);
+	static void string_to_avatar_url(const std::string&, SceNpAvatarUrl* avatar_url);
```

Comment:
string_view is not compatible here?

Reply:
It gets std::string reference anyway, plus there is some bug (in my STL?) that prevents string_view to work.